fragmentTypeFetch.js

```diff
@@ -0,0 +1,45 @@
+// https://www.apollographql.com/docs/react/advanced/fragments.html#fragment-matcher
+
+const fetch = require('node-fetch')
+const fs = require('fs')
+const SPACE_ID = 'qz0n5cdakyl9'
```

Comment:
These should be picked from the `process.env` I think, since you have the `.env` file already you can use something like [`dotenv`](https://www.npmjs.com/package/dotenv) and [preload](https://www.npmjs.com/package/dotenv#preload) them.

Reply:
done!